backend/app/pydantic_models/quiz.py

```
@@ -1,7 +1,13 @@
 from .question import Question
 from .multiselect_question import MultiSelectQuestion
 
 class Quiz(BaseModel):
-    questions: List[Union[Question, MultiSelectQuestion]]
+    ID: str
+    questions: List[Tuple[bool, Union[Question, MultiSelectQuestion]]]
+    score: int
+    notes: List[str]
+    type: str
+    playermode: str
+
```

Pylint (3.3.7)

[convention] Trailing newlines

(C0305)